### PR TITLE
Template for website reviewer nominations. [RELATED #61]

### PR DESCRIPTION
**Pull request by sptramer**
Feedback welcome to address the open question of how one becomes a reviewer of the website and how they maintain that status.

### CODE REVIEW

**Comment by willihay** [comment]
Any reason not to edit the [existing template](https://github.com/o3de/sig-docs-community/blob/main/.github/ISSUE_TEMPLATE/reviewer-maintainer-nomination.md) instead of creating a YAML file template?

File: .github/ISSUE_TEMPLATE/website-reviewer.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+labels: nomination/website-reviewer
+body:
+  - type: markdown
+    attributes:
+      value: |
+        The Documentation and Community SIG maintains several reviewer roles requiring SIG approval - this nomination form is for serving as a _website reviewer_, responsible for reviewing contributions that add layouts, templates, improve functionality, or change the appearance of o3de.org in some way. Responsibilities include:
+
+        * Review changes to layouts, partials, and any website javascript submissions.
+        
+        **Maintaing status:** <???>
```
Comment (willihay) [inline .github/ISSUE_TEMPLATE/website-reviewer.yaml:13]: How about:
```suggestion
        **Maintaining status:** Once engaged in a review, promptly reply to any follow-up questions regarding your feedback. SIG chairpersons can remove the approval of this role, at their discretion, from reviewers who consistently fail to reply to questions regarding their feedback, or who have been inactive for an extended period of time during which reviews have been requested of the group.
```

**Review by chanmosq — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/website-reviewer.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        The Documentation and Community SIG maintains several reviewer roles requiring SIG approval - this nomination form is for serving as a _website reviewer_, responsible for reviewing contributions that add layouts, templates, improve functionality, or change the appearance of [o3de.org](https://www.o3de.org/) in some way. Responsibilities include:
+
+        * Review changes to layouts, partials, and any website development submissions.
+        * Provide guidance on discussions surrounding website development, as needed.
+        * Advocate for quality changes to uphold the website's infrastructure.
+        **Maintaining status:** Serve in the reviewer rotation as required. Serving in the review rotation requires an initial review of the submission,
```
Comment (chanmosq) [inline .github/ISSUE_TEMPLATE/website-reviewer.yaml:14]: NIt: Add a space here
```suggestion

        **Maintaining status:** Serve in the reviewer rotation as required. Serving in the review rotation requires an initial review of the submission,
```

File: .github/ISSUE_TEMPLATE/website-reviewer.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+         as well as prompt follow-up responses to feedback questions. SIG chairpersons and maintainers will remove reviewers who don't meet this requirement on 
+        a regular basis.
+
+        For more information about the general requirements of reviewer roles, read our [Community Membership](https://github.com/o3de/community/blob/main/community-membership.md) information.
+  - type: input
+    id: username
+    attributes:
+      label: Nominee GitHub username
```
Comment (chanmosq) [inline .github/ISSUE_TEMPLATE/website-reviewer.yaml:22]: Suggested in other nomination forms. Recommend doing it here too for consistency
```suggestion
      label: Nominee's GitHub username
```

**Review by chanmosq — APPROVED** [review]
Looks good! Left a few small suggestions, nothing blocking. Approving.

**Pull request merged**